zerver/openapi/zulip.yaml

```diff
@@ -207,6 +207,8 @@ paths:
                     properties:
                       result: {}
                       msg: {}
+                      ignored_parameters_unsupported:
+                        $ref: "#/components/schemas/IgnoredParametersUnsupported"
```

Comment:
Could we have this be part of `JsonSuccessBase` instead of needing to include it in each instance?

Reply:
Done in the most recent update.